docs/man/ctags-lang-verilog.7.rst

```diff
@@ -98,12 +162,27 @@ with "--options=NONE --sort=no -o - --fields-SystemVerilog=+{parameter} input.sv
 	P3	input.sv	/^	parameter  P3 = "parameter";$/;"	c	module:no_parameter_port_list	parameter:
 	L7	input.sv	/^	localparam L7 = "localparam";$/;"	c	module:no_parameter_port_list
 
-Known Issues
----------------------------------------------------------------------
+TIPS
+~~~~
 
-See https://github.com/universal-ctags/ctags/issues/TBD.
+If you want to map files ``*.v`` to SystemVerilog, add
+``--langmap=SystemVerilog:.v`` option.
 
+KNOWN ISSUES
+---------------------------------------------------------------------
+
+- It assumes the input file is written in the correct grammar.  Otherwise output
+  of ctags is undefined.
```

Comment:
This is applicable to the most of all parsers. So I would like you to write this in ctags(1) instead of writing this in a parser.
I found an alternative sentence. "ctags implements garbage-in-garbage-out feature." :-P

Reply:
I did it.

Thank you. I learned new words, https://en.wikipedia.org/wiki/Garbage_in,_garbage_out.